### PR TITLE
Improve ACU scanning fault tolerance and logging

### PR DESCRIPTION
**Pull request by mhasself**
## Description

A few important things pertinent to constant elevation scanning:
- Limit the turnaround acceleration according to the maximum jerk constraint described in the SATP ICD.
- Log the parameters passed in by the user, and computed params.  For forensic tracking on failed scans.
- Monitor key program track error bits, and state them in the log if they fire during scan.
- Return False from the Process, if it appears to have exited due to error.
- Clear Faults before starting a scan (-- and also before a go_to).

## Motivation and Context

These are all to make real usage smoother, and forensic analysis possible (why didn't the scan several hours ago work?).  And the max jerk constraint basically came out because it was very easy to trigger a "Turnaround time too short" fault.

## How Has This Been Tested?

I developed this actively on SATP2.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by mhasself** [comment]
We will want this running on SATP1 before we next do overnight scans ...

File: socs/agents/acu/agent.py [hunk 1]
```diff
@@ -1076,6 +1076,10 @@ def go_to(self, session, params):
             if not acquired:
                 return False, f"Operation failed: {self.azel_lock.job} is running."
 
+            self.log.info('Clearing faults to prepare for motion.')
+            yield self.acu_control.clear_faults()
+            yield dsleep(1)
+
```
Comment (BrianJKoopman) [inline socs/agents/acu/agent.py:1082]: Should we have a similar block before boresight motion?

**Review by BrianJKoopman — COMMENTED** [review]
This looks good to me, just wondering if we should clear faults before boresight motion as well.

File: socs/agents/acu/agent.py (hunk 1 above)
Comment (mhasself) [reply socs/agents/acu/agent.py:1082]: yes, yes we should ...

**Review by BrianJKoopman — APPROVED** [review]
Great, thanks for adding. I'll merge once checks finish.

**Pull request merged**